bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -276,7 +274,7 @@ impl LockTime {
     }
 }
 
-impl_parse_str_through_int!(LockTime, from_consensus);
+impl_parse_str_from_int_infallible!(LockTime, u32, from_consensus);
```

Comment:
This adds boilerplate, why do you consider it better?

Reply:
I'm not sure I understand -- the macro used to only apply to cases where `u32` was the integer type, but now sometimes it's used for the `u64` case, so it needs to be extended.

Reply:
IIRC the additional method was optional and the type was inferred.

Reply:
Even if it can be inferred, having an explicit type here will improve readability.

Reply:
IIRC it can still be inferred for one set of the macros (fallible or infallible, I can't remember) but I thought it better to be uniform and explicit for both and the cost of extra boilerplate.

Reply:
Is this thread resolved @Kixunil or do you have concerns still that I need to address?

Reply:
I only dislike it slightly so I can accept it.

Reply:
Cool, cheers.